types/vtex.js/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/vtex.js",
-    "version": "2.11.9999",
+    "version": "3.0.0",
```

Comment:
the last version of vtex.js is 2.13. Anyway theres no need to change this if the PR changes isnt something of the new versions